CONTRIBUTING.md

```diff
@@ -158,10 +160,10 @@ Examples:
     qt: Add feed bump button
     log: Fix typo in log message
 
-The body of the pull request should contain enough description about what the
-patch does together with any justification/reasoning. You should include
-references to any discussions (for example other tickets or mailing list
-discussions).
+The body of the pull request should contain sufficient description of *what* the
+patch does, and even more importantly, *why*, with justification and reasoning.
+You should include references to any discussions (for example, other tickets or
```

Comment:
Nit in the original: per `git grep -i ticket`, this is the only occurrence of "ticket" in the repository.  OTOH, our documentation and the GitHub interface frequently use "issue".  Since this paragraph is being changed anyway, I suggest s/tickets/issues/.